src/app/mod.rs

```diff
@@ -188,7 +190,10 @@ impl App {
             )
             .await?;
         let xpub_id = xpub.id();
-        xpub.set_signer_config(config);
+        if let Some(key) = self.signer_encryption.key {
```

Comment:
This should never be None here .. need to ensure it is not none when config is loaded.